app/src/main/java/com/ncnf/map/ui/MapFragment.java

```diff
@@ -55,7 +55,7 @@
     private boolean map_ready = false;
 
     @Inject
-    EventProvider eventProvider;
+    PublicEventProvider publicEventProvider;
```

Comment:
Make public?

Reply:
This is also changed in the other PR. For this one, PublicEventProvider is still needed, because the source hasn't been modified yet